src/Common/src/System/Net/HttpKnownHeaderNames.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    internal static class HttpKnownHeaderNames
```

Comment:
Is there any reason to keep these constants internal? We end up all time duplicating this kind of constants, maybe we can make them public